aion/web/channels/room_channel.ex

```diff
@@ -92,14 +93,9 @@ defmodule Aion.RoomChannel do
       Logger.info("[channel] Question timed out in room: #{room_id}. Fetching a new one...")
 
       case old_state do
-        :question -> send_question_break(socket)
-        :break -> send_display_question(socket)
+        :question -> become_in_question_break_state(socket)
+        :break -> become_in_question_displayed_state(socket)
```

Comment:
`enter_question_break_state` and `enter_question_displayed_state`?

Reply:
Yeah, that sounds better, I took the `become` from Akka subconsciously :rofl: